src/Framework/Framework/Controls/KnockoutHelper.cs

```diff
@@ -355,8 +355,8 @@ private static JsExpression TransformOptionValueToExpression(DotvvmBindableObjec
             {
                 case IValueBinding binding: {
                     var adjustedCode = binding.GetParametrizedKnockoutExpression(handler, unwrapped: true).AssignParameters(o =>
-                        o == JavascriptTranslator.KnockoutContextParameter ? new ParametrizedCode("c") :
-                        o == JavascriptTranslator.KnockoutViewModelParameter ? new ParametrizedCode("d") :
+                        o == JavascriptTranslator.KnockoutContextParameter ? CodeParameterAssignment.FromIdentifier("c") :
```

Comment:
FYI: This is just a cosmetic change - the `FromIdentifier` correctly sets operator precedence, so the `c`/`d` variables don't get wrapped in unnecessary parenthesis.